postgres_copy/__init__.py

```diff
@@ -69,8 +76,23 @@ def copy_from(source, dest, engine, columns=(), **flags):
         formatted_flags,
     )
     cursor.copy_expert(copy, source)
-    conn.commit()
-    conn.close()
+    if autoclose:
+        conn.commit()
+        conn.close()
+
+def raw_connection_from(engine_or_conn):
+    """Extract a raw_connection and determine if it should be automatically closed.
+
+    Only connections opened by this package will be closed automatically.
+    """
+    autoclose = False
```

Comment:
Very minor, but this can be slightly more concise as

```python
if hasattr(engine_or_conn, 'cursor'):
    return engine_or_conn, False
if hasattr(engine_or_conn, 'connection'):
    return engine_or_conn.connection, False
return engine_or_conn.raw_connection(), True
```